share/functions/__fish_print_hostnames.fish

```diff
@@ -76,6 +76,11 @@ function __fish_print_hostnames -d "Print a list of known hostnames"
 
             set -l new_paths
             for path in $paths
+                # while ssh_config is using brackets to resolve env, they should be removed
+                # example
+                # in ssh_config: ${SOME_PATH}
+                # in fish:       $SOME_PATH
+                set path (string replace -r '\${([^}]+)}' '$1' $path)
```

Comment:
can we make it work for `${FOO}bar` too?
A possible fish syntax would be `"$FOO"bar`